index.html

```diff
@@ -64,128 +65,7 @@ <h3 class="intro2">LET’S CONNECT</h3>
-                    <div class="role">
-                        <div class="role-name">CANOPY</div>
-                        <div class="role-dot"></div>
-                        <div class="role-job">Back End Dev</div>
-                        <div class="role-dot"></div>
-                        <div class="role-year">2015</div>
-                    </div>
-                    <div class="project-details">
-                        <p>
-                            A daily selection of privately personalized
-                            reads; no accounts or sign-ups required.
-                        </p>
-                    </div>
-                    <ul class="skills">
-                        <li class="skill set1">html</li>
-                        <li class="skill set2">css</li>
-                        <li class="skill set3">javascript</li>
-                    </ul>
-                    <div class="button-click">
-                        <button type="button" class="btn">See Project</button>
-                    </div>
-                </div>
-            </div>
-
-            <div class="projects">
-                <div class="img-web">
-                    <img class="" src="images/project2.png" alt="project1">
-                </div>
-                <div class="project-description">
-                    <h1 class="title">Multi-Post Stories</h1>
-                    <div class="role">
-                        <div class="role-name">CANOPY</div>
-                        <div class="role-dot"></div>
-                        <div class="role-job">Back End Dev</div>
-                        <div class="role-dot"></div>
-                        <div class="role-year">2015</div>
-                    </div>
-                    <div class="project-details">
-                        <p>
-                            A daily selection of privately personalized
-                            reads; no accounts or sign-ups required.
-                        </p>
-                    </div>
-                    <ul class="skills">
-                        <li class="skill set1">html</li>
-                        <li class="skill set2">css</li>
-                        <li class="skill set3">javascript</li>
-                    </ul>
-                    <div class="button-click">
-                        <button type="button" class="btn">See Project</button>
-                    </div>
-                </div>
-            </div>
-
-            <div class="projects">
-                <div class="img-web">
-                    <img class="" src="images/project3.png" alt="project1">
-                </div>
-                <div class="project-description">
-                    <h1 class="title">Tonic</h1>
-                    <div class="role">
-                        <div class="role-name">CANOPY</div>
-                        <div class="role-dot"></div>
-                        <div class="role-job">Back End Dev</div>
```

Comment:
- [ ] As the last review said please make sure that your design matches the Figma design 
add  a blurred background to your pop window as in the Figma design, you said that the content of the project will be hidden so I don't see this as an issue hide the content of the portfolio when the user opens the popup and make it visible it when the user closes the popup

| Your design | Figma design |
| ------------- | --------------- |
|  ![image](https://user-images.githubusercontent.com/74744787/162028667-102dd044-db70-487a-b517-347194167006.png) | ![image](https://user-images.githubusercontent.com/74744787/162028805-8cb3272e-62e0-4f1d-83f0-487d354683f0.png) |

Reply:
@MeqdamAlqudah Hi, Thanks for the suggestion about the blurry part, once I enlarge the image this is what happens

![Screenshot (63)](https://user-images.githubusercontent.com/52131748/162039569-1a40b030-262f-449a-8ea8-f0e28e06e646.png)